invisible_cities/types/symbols.py

```diff
@@ -98,3 +99,14 @@ class SiPMCharge(AutoNameEnumBase):
 class WfType(AutoNameEnumBase):
     rwf  = auto()
     mcrd = auto()
+
+
+ALL_SYMBOLS = {}
+for etype in dict(locals()).values():
+    if not isinstance(etype, EnumMeta): continue
+    if etype is EnumMeta              : continue
+    if etype is Enum                  : continue
+    if etype is AutoNameEnumBase      : continue
+
+    for symbol in etype:
+        ALL_SYMBOLS[symbol.name] = symbol
```

Comment:
I don't like this implementation because 

+ it uses action at a distance,
+ takes everything in `locals()` and tries to exclude everything irrelevant (and the stability of this set makes me nervous in Python).

How about a decorator on the enum classes, that adds the class' symbols to `ALL_SYMBOLS`? Or it could be done with metaclasses.

But that might be more trouble than it's worth. This certainly should not further delay merging this PR!